### PR TITLE
Website: Update the FAQ page

### PR DESCRIPTION
**Pull request by AnnikaCodes**
Changes courtesy of Zodiax and Iyarito.

### CODE REVIEW

File: website/pages/faq.md [hunk 1]
```diff
@@ -139,21 +139,11 @@ If you want to challenge a specific user, click their username or click "Find a
...
 
 An endless battle is, as the name implies, a battle that cannot end. The common components of an endless battle are Pokemon that trap the target (via Block, Shadow Tag, or Arena Trap), moves that heal the opponent (usually Heal Pulse), ways to keep your own Pokemon healthy (Slack Off, Recover, or Roost), Leppa Berries to restore the user's lost PP, and continual generation of Leppa Berries (via Recycle).
-
-The opponent is forced to Struggle, and because Struggle can never run out of PP, the battle would go on forever since a player can continuously heal both Pokemon (and prevent them from fainting). Endless battles are banned - if you encounter a "strategy" like this, alert a staff member in the [Help room](https://psim.us/help).
+ 
+The opponent is forced to Struggle, and because Struggle can never run out of PP, the battle would go on forever since a player can continuously heal both Pokemon (and prevent them from fainting). Battles are capped at 1000 turns; if that number of turns is reached, the battle ends and results in a tie. If you encounter a "strategy" like this, alert a global staff member by either sending them a PM or with a help ticket using `/report`.
```
Comment (scheibo) [inline website/pages/faq.md:146]: Why direct people to report endless battles? The battle terminates after 1000 terms, they're not "endless". Suggest `/offertie` or filing a bug report, what is staff going to do?

Comment (DaWoblefet) [reply website/pages/faq.md:146]: Agreed, don't report endless battles.

Comment (AnnikaCodes) [reply website/pages/faq.md]: I talked to Zodiax (who sent me these changes) and we worked this out.

**Comment by Zarel** [comment]
Thanks.

**Pull request merged**